kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/kms/vault/KubeVaultTestKmsFacade.java

```diff
@@ -185,7 +165,7 @@ public void generateKek(String alias) {
 
         @Override
         public void deleteKek(String alias) {
-            throw new UnsupportedOperationException();
+            LOGGER.atWarn().log("KEK deletion is not supported in Vault");
```

Comment:
That's not true.   HashiCorp Vault does support key delete.  You just need to flip the https://developer.hashicorp.com/vault/api-docs/secret/transit#deletion_allowed flag first.  Just make a call to update before you call delete.

https://developer.hashicorp.com/vault/api-docs/secret/transit#update-key-configuration